client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useLayoutEffect } from 'react';
+import { Button, CircularProgress, IconButton, Typography, Skeleton, Grid } from '@mui/material';
+import { Box } from '@mui/system';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import { Formik, FormikHelpers } from 'formik';
+import AvatarDisplay from '../../../components/AvatarDisplay/AvatarDisplay';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import { makeStyles } from '@mui/styles';
+import editProfile from '../../../helpers/APICalls/editProfile';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    '& .MuiButton-root': {
+      textTransform: 'none',
+    },
+  },
+  displayWrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textAlign: 'center',
+  },
+  displayPic: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignContent: 'center',
+  },
+  displayText: {
+    maxWidth: 250,
+    padding: '10px',
+    color: '#969696',
+  },
+  uploadButton: {
+    height: 70,
+    textTransform: 'none',
+  },
+  deleteButton: {
+    color: 'black',
+  },
+});
+
+interface ProfilePhotoProps {
+  header: string;
+  currentUser?: User;
+  currentProfile?: any;
```

Comment:
Eventually we can use the Profile interface here instead of type `any` 👍🏽

Reply:
The Profile interface is created in integration branch, and I'm thinking to apply that in #52